patternfly/patternfly-elements
# fix: pfe-tabs vertical height

- Update pfe-tabs demo page so that it contains HTML header tags in the tab headers. 
- Add CSS so that tab-panels which are shorter than the tabs are still at least as tall as the tab headers.

## To test

Check the [demo page](https://deploy-preview-796--happy-galileo-ea79c4.netlify.com/elements/pfe-tabs/demo/), under the heading `Style: Vertical earth with custom tab typography & header tags`. Note that the first tab-panel has content which is very short, and yet the panel is at least as tall as the tabs on the left. 

The heading tags are just to show that pfe-tabs support HTML header markup.
